packages/itwinui-react/vitest.config.ts

```diff
@@ -2,15 +2,16 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-module.exports = new Proxy(
-  {},
-  {
-    get: (target, prop) => {
-      // instead of returning scoped css modules, we will preserve the original classes
-      if (prop.startsWith('iui')) {
-        return prop;
-      }
-      return Reflect.get(target, prop);
-    },
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: 'setupTests.ts',
+    exclude: ['/node_modules/', '/esm/', '/cjs/'],
+    reporters: ['junit', 'default'],
+    outputFile: 'coverage/junit.xml',
+    css: false,
   },
```

Comment:
Continuing conversation from https://github.com/iTwin/iTwinUI/pull/1784#discussion_r1467022585

Lets try using a vite plugin:

```suggestion
  },
  plugins: [
    {
      name: 'mock-css',
      resolveId(id) {
        if (id.endsWith('styles.css')) {
          return id;
        }
        return null;
      },
      load(id) {
        if (id.endsWith('styles.css')) {
          return '';
        }
        return null;
      },
    },
  ],
```